README.md

```diff
@@ -1,32 +1,31 @@
 # Mensinator🩸
 Private period tracking with custom symptoms. We don't track your data - you do. Period.
 
-## 🫂 Made for friends by friends
-Originally Emma made Mensinator for her friend with goals of better symptom tracking and privacy respect.  
-Soon, many of us joined and are now actively building the Mensinator together.  
-It is friendly, straightforward, and respects privacy.  
-We wanted an app to use, not an app to sell data to random corporations.
-
 ## 💡 Features
-- Period and ovulation tracking and statistics,
-- ***Customizable and colour-coded symptoms***,
-- Cycle prediction with average period calculation,
-- Cycle prediction with average ***luteal phase calculation for irregular cycles***,
-- Data export and import for backups or moving to another device,
-- No tracking. Your data is yours. No data is sent anywhere.
-- No bloat. No useless or complex features. No sign-up. No emails.
+- Period and ovulation tracking and statistics.
+- ***Customizable, colour-coded symptoms***.
+- Cycle prediction with average period calculation.
+- Cycle prediction with average ***luteal phase calculation for irregular cycles***.
+- Data export and import for backups or moving to another device.
+- No tracking. No sign-up. Your data is yours. No data is sent anywhere.
+- No bloat. No useless or complex features. No emails.
+
+## 🫂 Made for friends by friends
+Originally, Emma made Mensinator for a friend who was looking for an app that would track unique period symptoms and respect her privacy.
+Soon, more like-minded individuals joined the project. 
+We all wanted a user-friendly, straigtforward app made for us, not an app that uses our data for profit. We are now actively building Mensinator together. 
```

Comment:
I wrote friendly to imply comfort and humanity. User-friendly seems corporate and redundant as straigtforward already conveys similar information.

Reply:
I see what you're saying. I struggled to think of a better word, because we don't really use "friendly" in this way. Personally I wouldn't know what it means in an app description. Friendly is usually said about a living thing, so it might imply that there's cute characters or something like that in the app. If anyone has suggestions for other words I'm definitely open to changing it. You're right having both "user-friendly" and "straightforward" isn't the best.